poms/pom.xml

```diff
@@ -161,6 +162,18 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+                <plugin>
+                  <groupId>com.itemis.maven.plugins</groupId>
+                  <artifactId>unleash-maven-plugin</artifactId>
+                  <version>2.6.0</version>
```

Comment:
Given you added them to `pluginManagement` you should not need to repeat the versions here 2x 

Or if you can not reach the definition in the `pluginManagement` from here should it then be removed there? Because as far as I know defining a plugin only in the `pluginManagement` will not activate it.

PS: if these stay the version number comment of the other location applies here instead.

Reply:
Same answer as above. It does not inherit from openHAB-core pom. I fully agree with you that we currently have much redundancy in many poms and I hate it to do it like that. But good news: refactoring will become easy as soon as we have the automated release as a regression test ;-)

Reply:
P.S.: In both files it's defined only once within the pluginManagement section and it's sufficient to do so.

Reply:
Source: https://maven.apache.org/pom.html#Plugin_Management
> However, this only configures plugins that are actually referenced within the plugins element in the children.

Do you reference it somewhere else or externally then?

Reply:
No. I think the point here is that the plugin is not needed at build time, just it's goals have to be reachable via CLI. And for this reason, it's completely sufficient to put it to `<pluginManagement>`. It's the same like with the buildHelper plugin: You don't have to specify it within the `<plugins>` section because it does not modify the build behaviour, it just provides additional goals for CLI usage.

Reply:
With external I meant CLI, so yes :-)

Reply:
Ah OK, then yes ;-)